distributions/demo-server/src/main/resources/config/default_retry.yaml

```diff
@@ -0,0 +1,14 @@
+
+  strategyMappings:
+  - regexes:
+    - "status code: 4\\d\\d"
+    strategy: !<Fatal> {}
+  - regexes:
+    - "simple"
+    strategy: !<Regular>
```

Comment:
I wouldn't call it regular.

I think uniform or something like that would be better, as in reality exponential is much more prevalent.